src/app/features/shared/components/molecules/InfoHeading/InfoHeading.tsx

```diff
@@ -1,28 +1,18 @@
 import React from "react"
 import { Heading } from "@datapunt/asc-ui"
 import InfoButton from "./InfoButton"
-import Modal, { ModalBlock } from "app/features/shared/components/molecules/Modal/Modal"
-import { useModal } from "app/features/shared/components/molecules/Modal/hooks/useModal"
 
 type Props = {
   infoTitle: string
-  infoText: string
+  infoText: string | JSX.Element
   as?: "h1" | "h2" | "h3" | "h4" | "h5" | "h6"
-  children: string | JSX.Element
+  children?: string
```

Comment:
I think this should be `children: JSX.Element`

Reply:
Done